common/tools/add-identity-mgmt-lib.js

```diff
@@ -0,0 +1,229 @@
+const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
+
+// Create credentials using the \`@azure/identity\` package.
+// Please note that you can also use credentials from the \`@azure/ms-rest-nodeauth\` package instead.
+const creds = new DefaultAzureCredential();
+const client = new IotCentralClient(creds, subscriptionId);
+const resourceGroupName = "testresourceGroupName";
+const resourceName = "testresourceName";
+client.apps.get(resourceGroupName, resourceName).then((result) => {
+  console.log("The result is:");
+  console.log(result);
+}).catch((err) => {
+  console.log("An error occurred:");
+  console.error(err);
+});
+\`\`\`
+
+#### browser - Authentication, client creation, and get apps as an example written in JavaScript.
+
+In browser applications, we recommend using the \`InteractiveBrowserCredential\` that interactively authenticates using the default system browser.
+It is necessary to [create an Azure App Registration](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) in the portal for your web application first.
+
+##### Sample code
+
+- index.html
+
+\`\`\`html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>@azure/arm-iotcentral sample</title>
+    <script src="node_modules/@azure/ms-rest-azure-js/dist/msRestAzure.js"></script>
+    <script src="node_modules/@azure/identity/dist/index.js"></script>
+    <script src="node_modules/@azure/arm-iotcentral/dist/arm-iotcentral.js"></script>
+    <script type="text/javascript">
+      const subscriptionId = "<Subscription_Id>";
+      // Create credentials using the \`@azure/identity\` package.
+      // Please note that you can also use credentials from the \`@azure/ms-rest-browserauth\` package instead.
+      const credential = new InteractiveBrowserCredential(
+      {
+        clientId: "<client id for your Azure AD app>",
+        tenant: "<optional tenant for your organization>"
+      });
+      const client = new Azure.ArmIotcentral.IotCentralClient(creds, subscriptionId);
+      const resourceGroupName = "testresourceGroupName";
+      const resourceName = "testresourceName";
+      client.apps.get(resourceGroupName, resourceName).then((result) => {
+        console.log("The result is:");
+        console.log(result);
+      }).catch((err) => {
+        console.log("An error occurred:");
+        console.error(err);
+      });
+    </script>
+  </head>
+  <body></body>
+</html>
+\`\`\`
+
+## Related projects
+
+- [Microsoft Azure SDK for Javascript](https://github.com/Azure/azure-sdk-for-js)
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/iotcentral/arm-iotcentral/README.png)
+`;
+}
+
+function updatePackageJson(newPackageVersion) {
+  return function(content) {
+    return content
+      .replace(/"version": "\d+.\d+.\d+",/ms, `"version": "${newPackageVersion}",`)
+      .replace(/"@azure\/ms-rest-azure-js": ".+?"/ms, '"@azure/ms-rest-azure-js": "^2.1.0"')
```

Comment:
There will be some packages that use v1 of ms-rest-js and ms-rest-azure-js, they should be replaced with the corresponding v1 version that has the Azure Identity support

Reply:
Done.